diff --git a/dist/session.js b/dist/session.js
index 1b92b458d7ebe6555be516dad26b1abf40efc64e..ae64f5d7e2cc6d4a41f70d42e3529ff3945c0446 100644
--- a/dist/session.js
+++ b/dist/session.js
@@ -1,4 +1,4 @@
-import { ConnectionError, ResponseError } from "vscode-jsonrpc/node";
+import { ConnectionError, ResponseError } from "vscode-jsonrpc/node.js";
 import { createSessionRpc } from "./generated/rpc.js";
 class CopilotSession {
   /**
